DependencyInjection/DoctrineExtension.php

```diff
@@ -396,6 +396,14 @@ protected function ormLoad(array $config, ContainerBuilder $container)
                 $def->addTag('doctrine.event_subscriber');
             }
         }
+
+        if (!class_exists('Symfony\Bridge\Doctrine\PropertyInfo\DoctrineExtractor')) {
```

Comment:
you should check first for the PropertyInfo component, as the bridge does not force installing PropertyInfo (but we force installing the bridge)